src/Command/SequentialProcessQueueCommand.php

```diff
@@ -90,6 +90,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 // call the garbage collector to avoid too many connections & memory issue
                 if (($i + 1) % 200 === 0) {
```

Comment:
maybe `%200` is too high?

Reply:
maybe making it customize-able in a config setting and keep the 200 as default could be an idea for an improvement